src/client/view/mostUsedWords.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { SlidingSearchBar } from './searchBar';
+
+const WordInfo = ({ wordInfo }) => (

```

Comment:
It's not a word though, it's a word and a count, thus would be wrong to call it a Word